recipe/meta.yaml

```diff
@@ -43,7 +43,7 @@ requirements:
     - python
     - {{ pin_compatible('llvmlite', max_pin='x.x') }}
     - {{ pin_compatible('numpy') }}
-    - llvmlite >=0.30.0
+    - llvmlite >=0.31.0
```

Comment:
```suggestion
```
we already have the stricter `{{ pin_compatible('llvmlite', max_pin='x.x') }}` above

Reply:
(Will make this change on the next push, or if someone else does it, hopefully they will too)